supported_os_data.yml

```diff
@@ -33,17 +34,21 @@ shipping:
     '81': 10.6.6
     '91': 10.7.3
     '101': 10.7.4
+    '102': 10.8.2
     '111': 10.9.0
-    '113': 10.9.4
```

Comment:
Is this entry supposed to be removed?

Reply:
Yes, the old entry was incorrect. `MacBookPro11,3` was used for both the Late 2013 and Mid 2014 models. The former of which shipped with 10.9.0: https://everymac.com/ultimate-mac-lookup/?search_keywords=MacBookPro11,3